mappings/net/minecraft/entity/ai/goal/FleeEntityGoal.mapping

```diff
@@ -1 +1,24 @@
 CLASS alt net/minecraft/entity/ai/goal/FleeEntityGoal
+	FIELD a fleeingEntity Laip;
+	FIELD b targetEntity Laii;
+	FIELD c fleeDistance F
+	FIELD d fleePath Lcmr;
+	FIELD e fleeingEntityNavigation Laox;
+	FIELD i fleeSlowSpeed D
+	FIELD j fleeFastSpeed D
+	FIELD k fleeFromPredicate Lapp;
```

Comment:
Same, `flee` prefix is redundant as you are already in the `FleeEntityGoal` class